### PR TITLE
Added Scroll bar to filters for desktop users

### LINKED ISSUE #689: Ability to Scroll in Sidebar
## Problem Description
In the Desktop view, you cant scroll the sidebar. For example, the "All Sources" link has many options in it. To access the bottom options you should scroll the entire page. Also, the sidebar scrolls down when you scroll to the bottom of the page.

## Solution Description
Adding a separate scroll option to the sidebar solves both the issues

## Alternatives
None

## Additional context
If this issue is considered I would like to work on it

### PR DESCRIPTION
**Pull request by abhisheknaiidu**
## Fixes
Fixes #689  by @ManishBasavalinga

## Description
Added Scroll bar to filters for desktop users

## Screenshots

`DESKTOP VERSION`

scoll-desk

`MOBILE VERSION`

no-scroll-mob

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by abhisheknaiidu** [comment]
@akmadian @brenoferreira Is it fine with using a `npm` package or should I move forward with just CSS properties like `overflow-y` etc.

**Comment by abhisheknaiidu** [comment]
P.S-  I just checked it, and it's working fine with just adding two CSS properties `height: 100vh` and `overflow-y: scroll` on the top of div `wrapper`

File: package.json [hunk 1]
```diff
@@ -32,6 +32,9 @@
     "lodash.findindex": "^4.6.0",
     "lodash.isempty": "^4.4.0",
     "masonry-layout": "^4.2.2",
+    "overlayscrollbars": "^1.12.0",
```
Comment (brenoferreira) [inline package.json:35]: Please remove these npm packages. This should be fixed with CSS only

Comment (abhisheknaiidu) [reply package.json]: Done @brenoferreira

Comment (abhisheknaiidu) [reply package.json]: @brenoferreira it's working fine with just adding two CSS properties height: 100vh and `overflow-y: scroll` on the top of div wrapper
but it's not that robust as `npm` package!

Comment (brenoferreira) [reply package.json]: We don't need custom scrollbars nor anything else that justifies loading hundreds of KBs in extra NPM packages. That has a high cost for users loading the page

**Comment by abhisheknaiidu** [comment]
Looks this now :

wr

File: src/components/SearchGridFilter.vue [hunk 2]
```diff
@@ -129,4 +129,9 @@ export default {
   }
 }
 
+#content-wrapper {
```
Comment (brenoferreira) [inline src/components/SearchGridFilter.vue:132]: You can add this to the `__visible` [style](https://github.com/creativecommons/cccatalog-frontend/blob/527df3f30019e8bb2b0e9a5e901dce8cec9ffa62/src/components/SearchGridFilter.vue#L126). No need to add an id here

File: package.json (hunk 1 above)
Comment (abhisheknaiidu) [reply package.json]: Ohkk Noted!

File: src/components/SearchGridFilter.vue (hunk 2 above)
Comment (abhisheknaiidu) [reply src/components/SearchGridFilter.vue]: Done @brenoferreira

**Comment by abhisheknaiidu** [comment]
But @brenoferreira Its creating problem in Mobile Version, should I just again change it to only Desktop version?

**Pull request merged**